app/scripts/modules/amazon/src/loadBalancer/details/targetGroupDetails.html

```diff
@@ -0,0 +1,104 @@
+<div class="details-panel">
+
+  <div ng-if="ctrl.state.loading" class="header">
+    <div class="close-button">
+      <a class="btn btn-link"
+         ui-sref="^">
+        <span class="glyphicon glyphicon-remove"></span>
+      </a>
+    </div>
+    <h4 class="text-center">
+      <span us-spinner="{radius:20, width:6, length: 12}"></span>
+    </h4>
+  </div>
+
+  <div ng-if="!ctrl.state.loading" class="header">
+    <div class="close-button">
+      <a class="btn btn-link"
+         ui-sref="^">
+        <span class="glyphicon glyphicon-remove"></span>
+      </a>
+    </div>
+    <div class="header-text">
+      <span class="icon icon-elb"></span>
+      <h3 select-on-dbl-click>
+        {{ctrl.targetGroup.name}}
+        <render-if-feature feature="entityTags">
+          <!--<entity-ui-tags component="ctrl.loadBalancer"
+                          application="ctrl.application"
+                          entity-type="loadBalancer"
+                          page-location="details"
+                          on-update="ctrl.application.loadBalancers.refresh()"></entity-ui-tags>-->
+        </render-if-feature>
```

Comment:
one is `<entity-ui-tags>` and one is `<add-entity-ui-tag-links>` I'll remove them both; easy enough to copy over if/when we add entity tag support to target groups.